packages/firestore/test/unit/util/bundle.test.ts

```diff
@@ -61,15 +84,31 @@ function lengthPrefixedString(o: {}): string {
   });
 });
 
-// eslint-disable-next-line no-restricted-properties
-(isNode() ? describe.skip : describe)('Bundle ', () => {
+describe('Bundle ', () => {
   genericBundleReadingTests(1);
   genericBundleReadingTests(4);
   genericBundleReadingTests(64);
   genericBundleReadingTests(1024);
 });
 
 function genericBundleReadingTests(bytesPerRead: number): void {
+  // On Node, we need to override `bytesPerRead` from it's platform's `toByteStreamReader` call.
+  if (isNode()) {
+    const platform = PlatformSupport.getPlatform();
+    platform.toByteStreamReader = source =>
+      toByteStreamReader(source as Uint8Array, bytesPerRead);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (PlatformSupport as any)._forceSetPlatform(platform);
```

Comment:
I think you should try really hard to to have to do this. Options:

- Have a Node specific test file (bundle.node.test.ts similar to serializer.node.test.ts)
- Implement this logic in TestPlatform's `toByteStreamReader`.

A very strong argument against this pattern is that guaranteeing cleanup for this type of global state modification is hard. It even looks like you opted not do to it at all :)

Reply:
Actually making `toByteStreamReader` take a `bytesPerRead` eliminates the problem.